common/lib/xmodule/xmodule/js/fixtures/problem_content.html

```diff
@@ -23,7 +23,7 @@ <h3 class="hd hd-2 problem-header">Problem Header</h3>
         <button class="show btn-default btn-small"><span class="show-label">Show Answer(s)</span> <span class="sr">(for question(s) above - adjacent to each field)</span></button>
       </span>
     </div>
-    <button class="submit btn-brand" data-submitting="Submitting" data-value="Submit"><span class="submit-label">Submit</span><span class="sr"> your answer</span></button>
+    <button class="submit btn-brand" data-submitting="Submitting" data-value="Submit" data-should-enable-submit-button="True"><span class="submit-label">Submit</span><span class="sr"> your answer</span></button>
```

Comment:
For existing Jasmine tests, never disable the Submit button (which is the same behavior as previously).